.gitignore

```diff
@@ -1,10 +1,10 @@
-.idea/
-.project
+# Please do not use this ignore file to define platform specific files.
+#
+# For these purposes create a global .gitignore file, which is a list of rules
+# for ignoring files in every Git repository on your computer.
+#
+# https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
+
 composer.lock
-composer.phar
 vendor/
-data.txt
-cache/volt/*.php
```

Comment:
@sergeyklay Why are the cache folders removed from the gitignore?  These aren't platform specific rules but are actually rules for the Vokuro framework.  Once you run Vokuro it will fill these cache directories.